corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -365,8 +365,7 @@ def get_active_supply_points_data(domains, datespan, interval):
         num_active_supply_points = (CaseES()
                 .domain(domains)
                 .case_type(SUPPLY_POINT_CASE_TYPE)
-                .active_in_range(gte=start_date, lte=end_date)
-                .size(0)).run().hits
+                .active_in_range(gte=start_date, lte=end_date)).run().hits
```

Comment:
I think `size(0)` is correct actually. What should change is `hits` -> `total`.

`size` limits the documents that it returns, but we don't actually use any documents in this function. 
`hits` looks like it returns those documents.
`total` returns the number of documents that match the query which is what we want. 

Looks like I read the API incorrectly when I implemented this https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/es_query.py#L399-L418